src/services/review/RepoCheckerService.ts

```diff
@@ -293,6 +293,16 @@ export default class RepoCheckerService {
       .mapErr((error) => new SiteCheckerError(`${error}`))
   }
 
+  getResourceRoomPath(repoPath: string) {
+    const configPath = path.join(repoPath, "_config.yml")
+    const fileContent = fs.readFileSync(configPath, "utf8")
+    const resourceRoom = fileContent
+      .split("resources_name: ")[1]
+      .split("\n")[0]
+      .trim()
+    return resourceRoom
+  }
+
```

Comment:
tbh i think `PageService` has this already but it's fine ba

Reply:
[6a6fc29](https://github.com/isomerpages/isomercms-backend/pull/1187/commits/6a6fc296ed62a9292c47c50ad3dc0917908c8d85) the code to use page service instead